README.md

```diff
@@ -73,6 +73,7 @@ import visearch from 'visearch-javascript-sdk';
 // Set up keys
 visearch.set('app_key', 'YOUR_APP_KEY');
 visearch.set('tracker_code', 'YOUR_TRACKER_CODE');
+visearch.set('placement_id', 'YOUR_PLACEMENT_ID'); // required for products based API
```

Comment:
@Sisyphus25 , can we create separate Javascript i.e. productsearch instead of visearch. Mixing like this is confusing. Also, for Javascript we want to reduce the size as much as possible

Reply:
never mind, can ignore my comment, what you do make sense as well. however for widget, please create separate code folder and deploy to different CDN path.